spec/index.bs

```diff
@@ -1715,23 +1727,21 @@ dictionary GPUSamplerDescriptor : GPUObjectDescriptorBase {
 };
 </script>
 
-### {{GPUDevice}}.<dfn method for=GPUDevice>createSampler(descriptor)</dfn> ### {#sampler-createsampler}
-
-<div algorithm=GPUDevice.createSampler>
-    **Arguments:**
-        - optional {{GPUSamplerDescriptor}} |descriptor| = {}
-
-    **Returns:** {{GPUSampler}}
-
-    1. Let |s| be a new {{GPUSampler}} object.
-    1. Set the {{GPUSampler/[[compareEnable]]}} slot of |s| to false if the {{GPUSamplerDescriptor/compare}} attribute
-            of |descriptor| is null or undefined. Otherwise, set it to true.
-    1. Return |s|.
-
-    <div class=validusage dfn-for=GPUDevice.createSampler>
-        <dfn abstract-op>Valid Usage</dfn>
-    </div>
-</div>
+- {{GPUSamplerDescriptor/addressModeU}}, {{GPUSamplerDescriptor/addressModeV}}, 
+    and {{GPUSamplerDescriptor/addressModeW}} specify the address modes for the texture depth, width,
+    and height coordinates, respectively.
+- {{GPUSamplerDescriptor/magFilter}} specifies the sampling behavior for texture coordinates that do not
+    correspond to at least one texel.
+- {{GPUSamplerDescriptor/minFilter}} specifies the sampling behavior for texture coordinates that correspond
+    to multiple texels.
+- {{GPUSamplerDescriptor/mipmapFilter}} specifies behavior for sampling between two mipmap levels.
+- {{GPUSamplerDescriptor/lodMinClamp}} and {{GPUSamplerDescriptor/lodMaxClamp}} specify the minimum and
+    maximum levels of detail, respecitively, when sampling a texture.
```

Comment:
this may need to be clarified, as "maximum levels of detail used by internally when sampling a texture in the shader". The "internal" bit is important, to avoid confusion with user-provided LOD level in some of the sampling functions.